command/rootchain/supernet/stakemanager/params.go

```diff
@@ -30,5 +31,9 @@ func (s *stakeManagerDeployParams) validateFlags() error {
 	// validate jsonrpc address
 	_, err := helper.ParseJSONRPCAddress(s.jsonRPC)
 
+	if err := helper.ValidateProxyContractsAdmin(s.proxyContractsAdmin); err != nil {
+		return err
+	}
+
 	return err
```

Comment:
This part is incorrect, because we are not returning an error in case `helper.ParseJSONRPCAddress` fails

```suggestion
	_, err := helper.ParseJSONRPCAddress(s.jsonRPC)
    if err != nil {
        return err
    }

	return helper.ValidateProxyContractsAdmin(s.proxyContractsAdmin)
```

Reply:
Fixed 18372dcc66dbf29850da56b0164cc578f19bbdbc
It's strange that linter didn't catch this one.